onboard-server/src/main/java/no/vegvesen/ixn/serviceprovider/OnboardRestController.java

```diff
@@ -83,6 +87,37 @@ public ListCapabilitiesResponse listCapabilities(@PathVariable String servicePro
 		return response;
 	}
 
+	@RequestMapping(method = RequestMethod.GET, path = "/{serviceProviderName}/capabilities/fetch", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I think we need a better resource identifier than this. How about `/{serviceProviderName}/network/capabilities/` or something similar? Fetch is an operation, and a bad name for a resource.